src/FieldExchanger.jl

```diff
@@ -0,0 +1,210 @@
+"""
+    FieldExchanger
+
+This modules contains general functions for the exchange of fields between the
+atmospheric and surface component models.
+"""
+module FieldExchanger
+
+import SciMLBase: step!, reinit!
+export import_atmos_fields!,
+    import_combined_surface_fields!, update_sim!, update_model_sims!, reinit_model_sims!, step_model_sims!
+
+using ClimaCoupler: Interfacer, FluxCalculator, Regridder, Utilities
+
+"""
+    import_atmos_fields!(csf, model_sims, boundary_space, turbulent_fluxes)
+
+Updates the coupler with the atmospheric fluxes. The `Interfacer.get_field` functions
+(`:turbulent_energy_flux`, `:turbulent_moisture_flux`, `:radiative_energy_flux`, `:liquid_precipitation`, `:snow_precipitation`)
+have to be defined for the amtospheric component model type.
+
+# Arguments
+- `csf`: [NamedTuple] containing coupler fields.
+- `model_sims`: [NamedTuple] containing `ComponentModelSimulation`s.
+- `boundary_space`: [Spaces.AbstractSpace] the space of the coupler surface.
+- `turbulent_fluxes`: [TurbulentFluxPartition] denotes a flag for turbulent flux calculation.
+"""
+function import_atmos_fields!(csf, model_sims, boundary_space, turbulent_fluxes)
+    (; atmos_sim) = model_sims
+
+    # from atmos
+    if turbulent_fluxes == FluxCalculator.CombinedAtmosGrid()
+        Regridder.dummmy_remap!(csf.F_A, Interfacer.get_field(atmos_sim, Val(:turbulent_energy_flux)))
+        Regridder.dummmy_remap!(csf.F_E, Interfacer.get_field(atmos_sim, Val(:turbulent_moisture_flux)))
```

Comment:
The fields `F_A` and `F_E` in `csf` for turbulent energy and moisture fluxes aren't very clear to me. Can we rename those, maybe to something like `F_turb_energy`, `F_turb_moist`? There's definitely a tradeoff between name length and ease of understanding, but I think `F_A` and `F_E` are pretty opaque.

Reply:
Sure, I was sticking with the existing coupler names for now, but I'm all for more clarity! 💎